### ISSUE TITLE
Harmonise MICROBE soil metadata

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/microbe-central#5]
Now that we have most of the metadata from the first sampling round, we would like to harmonise the available metadata according to the standards we are producing.

**AI**:
- [x] Gather all the soil sample metadata currently available in BSD
- [x] Modify and document changes according to what has been discussed.
   - [x] CABI
   - [x] AIT
   - [x] HMGU
   - [x] DSMZ
   - [x] DSMZ (T0 samples - Grandparents)
   - [x] MUG
- [x] Create a checklist based on [ERC000022](https://www.ebi.ac.uk/ena/browser/view/ERC000022), including more mandatory fields
    - [ ] Decide where will it live

**Comment by ESapenaVentura** [comment]
**Changes**

### Refactor values
- Collection date: Removed "T00....Z" ending to timestamps - If the information is not available, we don't need the full timestamp.
- analysis date: Removed "T00....Z" ending to timestamps
- preservation temperature: refactored values to "degree Celsius" from "degree celsius" to align with [UO](https://www.ebi.ac.uk/ols4/ontologies/uo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FUO_0000005)
- deleted `sampleIdByYear` field in `HMGU` samples
- adding `ontologyTerms` tag to `soil type` field, using `ENVO` for the ontology.
- adding `ontologyTerms` tag to `time point` field, using `UO` for the ontology
- refactor `soil type` value to lowercase, to align with `ENVO` terms.

### Wrong fields (Probably artifacts in submission)
- deleted `biome.1` field in `HMGU` samples
- deleted `soil_type` and moved data to `soil type`

### Extra fields
- Deleted `year` and `plot_id_bexis` from HMGU samples (internal)

### Timepoint harmonisation
Basically, with the timepoints, we want to capture a few things; namely:
* How many days since the original sample was taken until the assay started
* How many days has the cryopreservation protocol been applied

For that, we originally assigned 2 fields; `collection date` and `time point`. However, these don't prove to capture all the essential data: for example, the first point above can't be fully known. Other things, like the date that the assay finishes, can be inferred from the data; however, I think it's good practice to have them explicitly stated. For that purpose, I am going to harmonise this information by doing the following changes:

- Added `assay start date`; capture the exact day the assay starts
- Added `assay finish date`: capture the exact day the assay finishes/is expected to finish
- Unmodified time point: basically, `assay finish date` - `assay start date`, but explicitly stated.
- Unmodified `collection date`, which is actually a GSC-approved field


### Preservation
Similar to the timepoints discussion, we have had other talks regarding temperatures for the whole assay; at the beginning, we were only capturing the `freezing method` and `preservation temperature`. However, after chatting with INRAE and CABI, we discovered that some fields were missing to capture the whole picture. With that in mind, I want to harmonise the sample metadata with the following fields:
- `freezing method` now to capture the method by using the ontologies provided with in [NCIT](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C84327?lang=en)
- Add `freezing temperature`, which would entail the temperature at which the sample was exposed with the method first. Learned from INRAE that they cool the samples in 2 steps: First they freeze them, to a certain temperature, until they're stabilised; once at that temperature (Usually -80), they can be preserved by further freezing in liquid nitrogen until -196 degrees.
- Add `cooling rate`, with either: the cooling rate (degrees celsius per minute) or uncontrolled (When the rate was not controlled)
- add `preservation method` and modify scope of `preservation temperature`, which is the method used to further freeze the sample during the assay.

**Comment by ESapenaVentura** [comment]
~On the **preservation** above:~
~- [ ] Before updating, send a message to people involved in soil samples, asking about the changes.~

New plan in last comment below

**Comment by ESapenaVentura** [comment]
Contacted AIT regarding non-consistent timepoints. In the meantime, the curation is paused - I need their input to push the changes

**Comment by ESapenaVentura** [comment]
Full checklist modified below:

[ERC000022_MICROBE_mod.json](https://github.com/user-attachments/files/17330233/ERC000022_MICROBE_mod.json)

List of changes:

## New fields
These fields are all **required**

- "biome",
- "assay start date",
- "assay finish date",
- "cryoprotectant",
- "freezing temperature",
- "freezing method",
- "preservation temperature",
- "preservation_method",
- "cooling_rate",
- "time point"

## Modified fields
- "soil type": Now validates against the ontology, instead of using an enum (Also, the enum is wrong; all soil types are capitalised and, in the ontology, they are not)

**Comment by ESapenaVentura** [comment]
Plan on metadata:
- [x] infer the timepoints and the new fields based on previous values and emails
- [x] Ask each centre to check their spreadsheets

**Comment by ESapenaVentura** [comment]
Sent the spreadsheets around (Forgot to update, whoops), waiting on responses. So far, we have received a response from AIT with a new question regarding if **we should drop the analysis date** now that we have the `assay finish date` field. We should probably discuss it in the DWG meeting

**Comment by ESapenaVentura** [comment]
Got another spreadsheet from Selma

**Comment by ESapenaVentura** [comment]
Got spreadsheets back from:
- CABI
- AIT
- DSMZ

Got a response from:
- HMGU

**Comment by ESapenaVentura** [comment]
## Sieving

On our discussion reviewing Conny's metadata ,we noticed that sieving needs to be looked at

## Units

Adding units from conny's metadata spreadsheet

**Comment by ESapenaVentura** [comment]
I am also adding lars' metadata from the T0 samples that needs to be added.

I have emailed Lars and sent him a merged spreadsheet with the already existing BioSamples metadata

**Comment by ESapenaVentura** [comment]
I am emailing the people that have not yet answered

I may take what we already have and update the metadata to have the fields and do another check for the catalogue

**Comment by ESapenaVentura** [comment]
I've finished (Aside from HMGU) the "first" round of harmonisation.

Before sending, I am creating a pydantic model based off the schema above and validating the samples.

**Comment by ESapenaVentura** [comment]
Validated and modified!
